twilio/twiml/voice_response.py

```diff
@@ -737,15 +737,19 @@ class Play(TwiML):
     """
     <Play> element
     """
-    def __init__(self, url, **kwargs):
+    def __init__(self, **kwargs):
         """
         Create a new <Play> element
 
-        :param url: media URL
-        :param kwargs: additional attributes
+        :param kwargs: attributes
         """
         super(Play, self).__init__(**kwargs)
-        self.value = url
+        # Set the value based on the optional url kwarg
```

Comment:
Try making `url` a named keyword argument so we don't have to pull it out manually.